### ISSUE TITLE
Child constructor should be disallowed when parent constructor is disallowed

### ISSUE THREAD

**Issue by spaze** [spaze/phpstan-disallowed-calls#146]
Given code like this (this is a code from a test actually):
```php
$sub = new Inheritance\Sub();
```

and configuration (from a test as well, so no neon syntax) like:
```php
'method' => 'Inheritance\Base::__construct()',
```

the call is not reported.

The call is reported if I change the configuration to
```php
'method' => 'Inheritance\Sub::__construct()',
```

**Comment by spaze** [comment]
The fix for this has been released in [2.11.2](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.11.2).
